### PR TITLE
Various minor changes

### PR DESCRIPTION
**Pull request by MykolasMaciekus**
-Removed useless comments 
-Removed useless CSS features
-Tried to improve mobile look of pagination a bit
-Moved some components out of main page components
-Made pagination prev and next logic more clear to read
-Some typo and grammar fixes

### CODE REVIEW

File: src/Components/CircularSpinner/CircularSpinner.tsx [hunk 1]
```diff
@@ -12,11 +12,11 @@ export const CircularSpinner = styled.div`
   -moz-animation: rotation 1s infinite linear;
   -o-animation: rotation 1s infinite linear;
   animation: rotation 1s infinite linear;
-  border: 6px solid rgba(0, 0, 0, .2);
+  border: 6px solid rgba(0, 0, 0, 0.2);
```
Comment (mdomasevicius) [inline src/Components/CircularSpinner/CircularSpinner.tsx:15]: how is `0.2` more brief than `.2` ? :)

File: src/Components/Inputs/Button.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const PRIMARY_CSS = css`
   background: ${themeCommon.colorKey};
 
   :active {
-    background: ${themeCommon.colorKey}BB;
```
Comment (mdomasevicius) [inline src/Components/Inputs/Button.tsx:20]: `alphaToHex(0.73)` ?

File: src/Components/Inputs/InputIconButton.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ interface IconButtonProps {
 
 const clickableCSS = css`
   :hover {
-    border: 1px solid ${themeCommon.colorDarkBlue}AA;
```
Comment (mdomasevicius) [inline src/Components/Inputs/InputIconButton.tsx:18]: `alphaToHex(0.67)`

File: src/Components/Pagination/Pagination.tsx [hunk 4]
```diff
@@ -53,20 +57,19 @@ export interface PaginationProps {
...
+  const onLastPage = currentPage + 1 >= totalPages
   return (
     <Container>
       <Button
         variant="outlined"
         label="Prev"
         rounded
         onClick={() => {
-          // Rethink else call
-          currentPage - 1 >= 1 && handlePageChange(currentPage - 1)
```
Comment (mdomasevicius) [inline src/Components/Pagination/Pagination.tsx:69]: don't need `{` and `}` for one liner arrow function

File: src/Pages/Authenticated/DashboardPage/LiveSessions/LiveSessions.tsx [hunk 5]
```diff
@@ -29,7 +29,7 @@ const { seconds2Time } = dates
 const { countryName } = location
 const { format, add } = bytes
 const { isDesktopQuery } = media
-// TODO: Move these to commons
+// TODO Rethink how to show full session/have full session copyable
```
Comment (mdomasevicius) [inline src/Pages/Authenticated/DashboardPage/LiveSessions/LiveSessions.tsx:32]: much boyscout, wow change, such effort :))

Comment (MykolasMaciekus) [reply src/Pages/Authenticated/DashboardPage/LiveSessions/LiveSessions.tsx:32]: We don't have a clear solution for this and I want to discuss how this should be implemented and how it should look before doing the change

File: src/Components/CircularSpinner/CircularSpinner.tsx (hunk 1 above)
Comment (MykolasMaciekus) [reply src/Components/CircularSpinner/CircularSpinner.tsx:15]: IDE auto-formatted

File: src/Pages/Authenticated/DashboardPage/LiveSessions/LiveSessions.tsx (hunk 5 above)
Comment (mdomasevicius) [reply src/Pages/Authenticated/DashboardPage/LiveSessions/LiveSessions.tsx:32]: trolling :)

**Review by mdomasevicius — APPROVED** [review]

**Pull request merged**